packages/engine/lib/execution/src/package/simulation/state/behavior_execution/reset_index_col.rs

```diff
@@ -16,16 +17,14 @@ pub struct ResetIndexCol {
 
 impl IntoArrowChange for ResetIndexCol {
     fn get_arrow_change(&self, range: std::ops::Range<usize>) -> memory::Result<ColumnChange> {
+        trace!("resetting index column");
```

Comment:
How verbose is this line? I don't mind keeping it, just want to flag that currently we misusing `trace` in other places, where `trace!` actually should be `debug!`. I just worry that this is too verbose to run everything with `RUST_LOG=trace`.